host_core/lib/host_core/host.ex

```diff
@@ -67,6 +67,12 @@ defmodule HostCore.Host do
       Logger.warn(warning)
     end
 
+    if Application.get_env(:host_core, :retain_rpc_subscriptions, false) do
+      Logger.warn(
+        "RPC Subscription Retention Enabled. This host instance is NOT stable in production!"
+      )
+    end
```

Comment:
Is this for a future update? I'm a little confused for the reasoning behind retaining an RPC subscription

Reply:
nvm, saw latest commit